src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -21,8 +21,7 @@
 
 <template>
 	<div class="wrapper">
-		<NcButton slot="trigger"
-			v-tooltip.bottom="t('spreed', 'Create a new group conversation')"
```

Comment:
Note: `slot="trigger"` is just removed here. This button is not passed to any of the component's slots.